### ISSUE TITLE
Show tolerance differently in last-box-filled bar 

### ISSUE THREAD

**Issue by wvengen** [foodcoop-adam/foodsoft#171]
The progress bar on the member ordering page that indicates how much the remaining box is filled, currently shows the sum of quantity and tolerance. And so you can't see if it is needed to order more to fill the remaining box, or if it's just extra tolerance that we have.

![foodsoft_-_my_current_order_-_2015-10-16_16 28 40](https://cloud.githubusercontent.com/assets/503804/10544343/201c6f8a-7423-11e5-84af-6e373a3cb4d1.png)

Proposal: split the missing_units into quantity and tolerance, and show tolerance grey-blue. Or perhaps use a blue-like color when there is quantity in missing_units, and use a grey-like color when there isn't.
Bottom line is that the attention of a member needs to be attracted when there action is needed.

**Comment by wvengen** [comment]
What about this? The first is an article where there's 2 missing units, and 3 total tolerance. This means that 2 more units need to be ordered for the box to become full. (You may notice that the progress bar shows 4 tolerance, which means that there's another member who only ordered 1 tolerance.)
![progress01](https://cloud.githubusercontent.com/assets/503804/10546729/0566291c-7430-11e5-8fb6-a5f1da2c7893.png)

The next is an article where there is no shortage, and there are still 7 units available in tolerance (by this member, as you can see), meaning that one member can still order 3 and have a full box.
![progress02](https://cloud.githubusercontent.com/assets/503804/10546848/e2d0ae6c-7430-11e5-88c6-b6b276906326.png)

I hope this motivates people to order more when it's useful, while providing a hint that one can still order more without having to order a full box. @tg-x @nhoening is this clear enough?
([explanation of the ordering algorithm in German](https://github.com/foodcoops/foodsoft/blob/master/doc/BESTELLEN.md))

**Comment by wvengen** [comment]
Implemented in #172.
